posts/static/makepost/posts.js

```diff
@@ -1,15 +1,36 @@
 function addPrivate() {
+    // structure reference for each input box
+    // div
+    //     privateDiv
+    //         removeButtonDiv
+    //             removeButton
+    //         option
+    
     let div = document.getElementById("append");
     let privateDiv = document.createElement('div');
+    let removeButtonDiv = document.createElement("div");
+    let removeButton = document.createElement("button");
     let option = document.createElement("input");
     option.classList.add('form-control');
     option.name = "visibleTo[]";
     option.value = "";
+
+    removeButtonDiv.className = "input-group-append";
+    removeButton.className = "btn btn-danger";
+    removeButton.onclick = deleteParent;
+    removeButtonDiv.appendChild(removeButton);
+
+    privateDiv.className = "input-group";
     privateDiv.appendChild(option);
+    privateDiv.appendChild(removeButtonDiv);
 
     div.appendChild(privateDiv);
 }
 
+function deleteParent() {
+    this.parentNode.parentNode.remove()
+}
+
 // not working, cant handle redirects
```

Comment:
Is this comment just left over? can we delete it or does it not handle redirects? I'm confused what it's saying

Reply:
Yeah this should be removed

Reply:
yeah idk, it was there when I arrived, so it remained after I left lol